configuration_manager/xml_parser.py

```diff
@@ -0,0 +1,104 @@
+import xml.etree.ElementTree as ET
+import errno
+import os
+
+
+class Parser(object):
```

Comment:
Nicely implemented and commented. I understood the logic instantly!

Reply:
Thanks!